### DISCUSSION TITLE
What type of picture do I need to train lora?

### DISCUSSION THREAD

**Discussion by vansaj0701** [lllyasviel/Fooocus#3439] [Q&A] [answered]
What type of picture and how many picture do I need to train lora for consistent' face and body of my influencer? How can I train my model/Lora/ Can I use lora to generate videos also?

**Comment by mashb1t** [comment, marked as answer]
> What type of picture and how many picture do I need to train lora for consistent' face and body of my influencer? 

Any kind of picture supported by the training software (most likely jpg/jpeg, png). You can do it with one image, but i'd recommend min. 20 images.

> How can I train my model/Lora/ Can I use lora to generate videos also?

I'd suggest https://github.com/bmaltais/kohya_ss if you have a potent GPU with min. 12GB VRAM (better 16GB), but you can also train a LoRA on civitai.com (more specific instruuuctions in https://education.civitai.com/using-civitai-the-on-site-lora-trainer/).

**Reply by vansaj0701** [reply]
Is there any Collab to train LoRa free of cost?

**Comment by lgwjames** [comment]
i sent a guy on fiver 30 images for £25 and came back a few hours later

**Reply by EvilBT** [reply]
So how does this lora work?

**Reply by EvilBT** [reply]
这个Lora的效果如何，有没有达到你的预期
